static/js/settings_org.js

```diff
@@ -868,6 +868,32 @@ function _set_up() {
     }
     realm_icon.build_realm_icon_widget(upload_realm_icon);
 
+    $('#deactivate_realm_button').on('click', function (e) {
+        if (!overlays.is_modal_open()) {
+            e.preventDefault();
+            e.stopPropagation();
+            overlays.open_modal('deactivate-realm-modal');
+        }
+    });
+
+    $('#do_deactivate_realm_button').on('click', function () {
+        if (overlays.is_modal_open()) {
+            overlays.close_active_modal();
+        }
+        channel.del({
+            url:'/json/realm/deactivate_realm',
+            success: function () {
+                window.location.href = "/accounts/deactivated/";
+            },
+            error: function (xhr) {
+                ui_report.error(
+                    i18n.t("Error deactivating realm"), xhr,
```

Comment:
I think this can just be "Failed: " + whatever error is returned from the backend.